packages/aws-cdk/test/api/deployments/deploy-stack.test.ts

```diff
@@ -391,35 +393,6 @@ test('reuse previous parameters if requested', async () => {
   } as CreateChangeSetCommandInput);
 });
 
-describe('ci=true', () => {
```

Comment:
deleted since this test was just testing that the CI dependent stream selection works.